### ISSUE TITLE
docker image gdal install error

### ISSUE THREAD

**Issue by jreiberkyle** [planetlabs/notebooks#71]
When the docker image is built, the following error occurs:

```
Step 2/12 : RUN conda install -y -c conda-forge gdal=2.3.1
 ---> Running in fc89ad22750d
Fetching package metadata .............

PackageNotFoundError: Packages missing in current channels:
            
  - gdal 2.3.1*
```

It looks like the gdal version needs to be updated

**Comment by jreiberkyle** [comment]
The [conda files page](https://anaconda.org/conda-forge/gdal/files) shows that only gdal version 2.4.0 is available. Gdal [release notes](https://trac.osgeo.org/gdal/wiki/Release/2.4.0-News) shows that 2.4.0 should be backward compatible with 2.3.1.

Therefore, we should be able to just update the package version in the Dockerfile and run a few notebooks to test.

**Comment by jreiberkyle** [comment]
Gdal version updated in #72. Still needs to be tested but blocked on data api issue that is erroring out the downloads.

**Comment by manivasaganrama** [comment]
@sarasafavi 

**While executing this command from your readme doc from this link:** https://github.com/planetlabs/notebooks
docker build --rm -t planet-notebooks .

**I am getting this error in Linux Bionic 18.04 LTS Terminal:** 
PackageNotFoundError: Packages missing in current channels:
            
  - gdal 2.4.0*

We have searched for the packages in the following channels:
            
  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.continuum.io/pkgs/main/linux-64
  - https://repo.continuum.io/pkgs/main/noarch
  - https://repo.continuum.io/pkgs/free/linux-64
  - https://repo.continuum.io/pkgs/free/noarch
  - https://repo.continuum.io/pkgs/r/linux-64
  - https://repo.continuum.io/pkgs/r/noarch
  - https://repo.continuum.io/pkgs/pro/linux-64
  - https://repo.continuum.io/pkgs/pro/noarch

**My system configuration:**
NAME="Ubuntu"
VERSION="18.04.2 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.2 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

**Comment by manivasaganrama** [comment]
> @sarasafavi
> @jreiberkyle
> @danabauer
> @nateweisiger
> @MattFerraro
> @theTechie
> @geowurster
> @spencermathews
> @jakedahn
> @gitter-badger
> 
> **While executing this command from your readme doc from this link:** https://github.com/planetlabs/notebooks
> docker build --rm -t planet-notebooks .
> 
> **I am getting this error in Linux Bionic 18.04 LTS Terminal:**
> PackageNotFoundError: Packages missing in current channels:
> 
> * gdal 2.4.0*
> 
> We have searched for the packages in the following channels:
> 
> * https://conda.anaconda.org/conda-forge/linux-64
> * https://conda.anaconda.org/conda-forge/noarch
> * https://repo.continuum.io/pkgs/main/linux-64
> * https://repo.continuum.io/pkgs/main/noarch
> * https://repo.continuum.io/pkgs/free/linux-64
> * https://repo.continuum.io/pkgs/free/noarch
> * https://repo.continuum.io/pkgs/r/linux-64
> * https://repo.continuum.io/pkgs/r/noarch
> * https://repo.continuum.io/pkgs/pro/linux-64
> * https://repo.continuum.io/pkgs/pro/noarch
> 
> **My system configuration:**
> NAME="Ubuntu"
> VERSION="18.04.2 LTS (Bionic Beaver)"
> ID=ubuntu
> ID_LIKE=debian
> PRETTY_NAME="Ubuntu 18.04.2 LTS"
> VERSION_ID="18.04"
> HOME_URL="https://www.ubuntu.com/"
> SUPPORT_URL="https://help.ubuntu.com/"
> BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
> PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
> VERSION_CODENAME=bionic
> UBUNTU_CODENAME=bionic

Issue got resolved when I updated the GDAL version to 3.0.1 by looking into the anaconda.org site.

Thanks for the support guys

**Comment by manivasaganrama** [comment]
**Issue got resolved when I updated the GDAL version to 3.0.1 by looking into the anaconda.org site.

Thanks for the support guys**
